storage/storage.go

```diff
@@ -35,3 +52,14 @@ func (f Fanout) Append(s *model.Sample) {
 		a.Append(s)
 	}
 }
+
+// Throttled returns true if at least one of the SampleAppenders in the Fanout
+// slice is throttled.
+func (f Fanout) Throttled() bool {
```

Comment:
I don't think that's the correct behavior. We are using the fanout to write to multiple storages simultaneously. We should not stop the world because one of potentially many is too busy.

In the current state it might work, because the remote storage always returns false and is slower anyway.
But this method should return true iff all sub-appenders are throttled.

On `Append()` it should then only call `Append()` on those sub-appenders that are not busy.